sunpy/net/helioviewer.py

```diff
@@ -88,7 +79,7 @@ def get_closest_image(self, date, **kwargs):
 
         return response
 
-    def download_jp2(self, date, directory=None, overwrite=False, **kwargs):
+    def download_jp2(self, date, sourceId, directory=None, overwrite=False, **kwargs):
```

Comment:
I am not sure we want `sourceId` to have a capital I. I am not sure that follows normal python naming convention. I would have to check.

Reply:
Oh yeah I guess I should change that capital I to a small one as it is indistinguishable from small L